doc/ref_cert/lfn/chapters/chapter10.md

```diff
@@ -5,16 +5,23 @@
 
 ## Table of Contents
 * [10.1 Introduction](#10.1)
-* [10.2 Heading](#10.2)
-* [10.3 Heading](#10.3)
+* [10.2 RM/RA-1 Requirements](#10.2)
+* [10.3 TC Mapping to Requirements](#10.3)
 
 <a name="10.1"></a>
 ## 10.1 Introduction
 
+Define, and describe the purpose of this chapter to be:
+
+- Define RM/RA-1 Openstack requirements
+- Map Framework to Requirements
 
 <a name="10.2"></a>
-## 10.2 Heading
+## 10.2 RM/RA-1 Requirements
 
+- Need to list out requirements from RM/RA-1.  
```

Comment:
in here we need to list out reference to Test Cases Requirement (coming from Chapter 6) and map them to real project test scripts.

Reply:
@rabi-abdel - updated verbiage for 10.2 to reflect the need/ask and intent for this section.  @sparayil and @collivier , any questions?